server/controllers/Investment.js

```diff
@@ -29,3 +29,20 @@ exports.addInvestment = async (req, res) => {
     });
   }
 };
+
+exports.getInvestment = async (req, res) => {
+  const { id } = req.params;
+
+  Investment.findById(id)
+    // .populate('user')
```

Comment:
will you need user?

Reply:
I don't think we will actually need this method since we only get investments through the user and the projects. I don't think we will have an option where we will be looking through all the investments or any individual investment so I will be removing this

Reply:
Alrighty. Can always add it back if the occasion arises.